src/proof/proof_rule.h

```diff
@@ -219,7 +219,7 @@ enum class PfRule : uint32_t
   THEORY_REWRITE,
   // The remaining rules in this section have the signature of a "trusted rule":
   //
-  // Children: none
+  // Children: ?
```

Comment:
clarify this. I think it'd be good to say that unless otherwise specified in the description below, the rule takes no children.